utils/dataset_manifest/utils.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (C) 2021 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+import hashlib
+import cv2 as cv
+from av import VideoFrame
+
+def rotate_image(image, angle):
```

Comment:
PIL and OpenCV has rotate function. Why should we implement our own version?
Do we really need to rotate on an arbitrary angle or 90, 180 degrees only?

Reply:
Similar to the comment above. 
It was implemented in [PR](https://github.com/openvinotoolkit/cvat/pull/2218) and do I really have to change it in this PR?